spellbook/models/uniswap/ethereum/uniswap_v1_ethereum_trades.sql

```diff
@@ -0,0 +1,105 @@
+{{ config(
+    schema = 'uniswap_v1_ethereum',
+    alias = 'trades',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = 'unique_trade_id'
+    )
+}}
+WITH dexs AS
+(
+        -- Uniswap v1 TokenPurchase
+    SELECT
+        t.evt_block_time AS block_time
```

Comment:
Lets add a block_date field and then add that as a partition key in the config

Reply:
definitely interested in adding and seeing this in action. will push a commit and resolve this